### ISSUE TITLE
Feature request: add local playlists

### ISSUE THREAD

**Issue by cooolgamer** [windows-server-2003/ThirdTube#109]
Hi! I really love this app and i'm using it to listen to music, and that's really nice that it still play on sleep mode.
But is it possible    add local playlists? I'm sure it is by using watch history and playlist system.
Because i'm always using watch history to chose a music.
Thanks

**Comment by CSab6482** [comment]
While this isn't exactly a solution to your issue, and I support this feature request, you may be interested in [LimePlayer3DS](https://github.com/oreo639/LimePlayer3DS), which plays music files on the 3DS. However, you do need to supply your own files instead of having the streaming convenience of YouTube, though `.pls` and `.m3u` playlist files are supported.

**Comment by cooolgamer** [comment]
yes I already know this but the problem is that I listen them directly on youtube and like you said, it require to download them, and 3ds sound do the job for that tbh
